conf/logging.cfg

```diff
@@ -1,5 +1,5 @@
 [handlers]
-keys=consoleHandler
+keys=consoleHandler, fileHandler
```

Comment:
I think fileHandler by default sometimes inconvenient...
We don't see the file for debug or troubleshoot...
Is there any reason for fileHandler by default?